From 0e2c56260a3ccddcccb5daa0c8aac3c6b92214ac Mon Sep 17 00:00:00 2001
From: tengjinchung <jin.chung.teng@intel.com>
Date: Thu, 17 Jun 2021 14:43:48 +0800
Subject: [PATCH] Corrected the install path

Fix CMake install path of libvpl_wayland.so to
$LIBDIR

Upstream-Status: Pending
inner-source pr #133

Signed-off-by: jinchung <jin.chung.teng@intel.com>
Signed-off-by: Yew, Chang Ching <chang.ching.yew@intel.com>
---
 tools/legacy/sample_common/src/vaapi_utils.cpp  | 2 +-
 tools/legacy/sample_misc/wayland/CMakeLists.txt | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/tools/legacy/sample_common/src/vaapi_utils.cpp b/tools/legacy/sample_common/src/vaapi_utils.cpp
index 833b4c1..23edf86 100644
--- a/tools/legacy/sample_common/src/vaapi_utils.cpp
+++ b/tools/legacy/sample_common/src/vaapi_utils.cpp
@@ -163,7 +163,7 @@ Xcbpresent_Proxy::~Xcbpresent_Proxy() {}
     #if defined(LIBVA_WAYLAND_SUPPORT)

 VA_WaylandClientProxy::VA_WaylandClientProxy()
-        : lib("libmfx_wayland.so"),
+        : lib("libvpl_wayland.so"),
           SIMPLE_LOADER_FUNCTION(WaylandCreate) {}

 VA_WaylandClientProxy::~VA_WaylandClientProxy() {}
diff --git a/tools/legacy/sample_misc/wayland/CMakeLists.txt b/tools/legacy/sample_misc/wayland/CMakeLists.txt
index 95057d4..90b55ca 100644
--- a/tools/legacy/sample_misc/wayland/CMakeLists.txt
+++ b/tools/legacy/sample_misc/wayland/CMakeLists.txt
@@ -2,7 +2,7 @@ if (NOT PKG_WAYLAND_CLIENT_FOUND)
     return()
 endif()

-set(TARGET mfx_wayland)
+set(TARGET vpl_wayland)

 find_package(PkgConfig REQUIRED)
 pkg_check_modules(PKG_WAYLAND_CLIENT wayland-client)
@@ -26,4 +26,4 @@ target_sources(${TARGET} PRIVATE

 target_link_libraries(${TARGET} sample_common wayland-client va drm drm_intel)

-install( TARGETS ${TARGETS} LIBRARY DESTINATION ${CMAKE_INSTALL_BINDIR} )
+install( TARGETS ${TARGET} LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR} )
--
2.32.0

